### ISSUE TITLE
Remove balloon/storage package

### ISSUE THREAD

**Issue by gdiazlo** [BBVA/qed#16]
Remove balloon/storage package and move its functionality to a balloon/storage.go inside ballon package.

We don't need a package there anymore, as the interface implementations don't need to be aware of it.

**Comment by panchoh** [comment]
It seems that an import loop is generated when trying to accomplish this:
```
▶ go build -v .
import cycle not allowed
package github.com/bbva/qed
	imports github.com/bbva/qed/cmd
	imports github.com/bbva/qed/api/apihttp
	imports github.com/bbva/qed/balloon
	imports github.com/bbva/qed/balloon/history
	imports github.com/bbva/qed/balloon
```
After discussing the issue with @gdiazlo and @aalda, we think we should explore moving a copy of the ```balloon/storage``` package to *each* of the trees, even though it means to duplicate it.  It allows some simplification, too, since some methods are not required by both trees.

**Comment by gdiazlo** [comment]
I'll leave this isse open, until we come up with a better design, or surrender.

**Comment by panchoh** [comment]
Ok.  I'll unassign myself from it, since it exceeds my current abilities.

**Comment by panchoh** [comment]
In case it helps: http://mantish.com/post/dealing-with-import-cycle-go/

One of the commenters suggests using a different package in the tests than the one being tested, to solve the issue.  Other points out that that precludes testing all but the public methods and structs, and that the problem is symptomatic of the tests being more functional-oriented than unit tests.

**Comment by gdiazlo** [comment]
I agree, I think its mostly due to our tests...see #1 :), the coming refactor will solve some of this, not sure the extent until we finish it.

**Comment by iknite** [comment]
Closed in #17
